packages/web/hooks/limit-orders/use-orderbook-pool.ts

```diff
@@ -1,15 +1,43 @@
+import { Dec } from "@keplr-wallet/unit";
+import { useMemo, useState } from "react";
+
+import { api } from "~/utils/trpc";
+
 export const useOrderbookPool = ({
   baseDenom,
   quoteDenom,
 }: {
   baseDenom: string;
   quoteDenom: string;
 }) => {
+  const [contractAddress] = useState<string>(
+    "osmo1svmdh0ega4jg44xc3gg36tkjpzrzlrgajv6v6c2wf0ul8m3gjajs0dps9w"
```

Comment:
This will eventually come from some config or registry value?

Reply:
I've started moving this already on [this](https://github.com/osmosis-labs/osmosis-frontend/tree/connor/use-orderbook-hook) branch